### PR TITLE
Update frontend.py

### PR DESCRIPTION
**Pull request by lindaassefa**
updated the tagging functionality in the frontend

### CODE REVIEW

**Review by nherman71 — COMMENTED** [review]
This seems to fix all errors, LGTM, going to wait on at least 1 more review before merging for safety.

File: frontend.py
Comment (augustus-thomas) [inline frontend.py:233]: why did you delete these?

**Review by augustus-thomas — CHANGES_REQUESTED** [review]
Thank you for this commit! Can you write some documentation on what you changed in this commit? I am unsure why some lines and functionality were deleted, and it would be helpful if we could know how this change will affect the codebase.

File: frontend.py
Comment (lindaassefa) [reply frontend.py:233]: These blocks were removed because a separate functionality for managing tags (adding, deleting, renaming, etc.) was introduced through a new state case 16 and the printTagMenu() function.The printTagMenu() function was introduced, which displays a menu for tag management operations. This menu allows users to add, delete, list, search, and rename tags without being tied to a specific note.(not much change actually)

File: frontend.py
Comment (Chrisvann1) [inline frontend.py:1]: I think this looks good to me. I am just slightly confused because I am still not 100% how the frontend functions so I  want someone else to make sure I am correct and merge it lol.

**Review by Chrisvann1 — COMMENTED** [review]
Could you possibly . I think the first line being empty is what is causing the pr to look like this and it is difficult to see what is happening.

**Comment by Chrisvann1** [comment]
I think this looks good to me. I am just slightly confused because I am still not 100% how the frontend functions so I  want someone else to make sure I am correct and merge it lol.

**Pull request merged**
